core/src/test/java/org/elasticsearch/action/admin/indices/TransportAnalyzeActionTests.java

```diff
@@ -276,6 +286,16 @@ public void testNonPreBuildTokenFilter() throws IOException {
         int default_bucket_size = 512;
         int default_hash_set_size = 1;
         assertEquals(default_hash_count * default_bucket_size * default_hash_set_size, tokens.size());
+    }
+
+    public void testNormalizerWithIndex() throws IOException {
+        AnalyzeRequest request = new AnalyzeRequest("index");
+        request.normalizer("my_normalizer");
+        request.text("ABc");
+        AnalyzeResponse analyze = TransportAnalyzeAction.analyze(request, AllFieldMapper.NAME, null, indexAnalyzers, registry, environment);
+        List<AnalyzeResponse.AnalyzeToken> tokens = analyze.getTokens();
 
+        assertEquals(1, tokens.size());
+        assertEquals("abc", tokens.get(0).getTerm());
```

Comment:
Ah, I added that test case in rest api test. Now, we are moving to filter/char_filter to analysis-common module, so I think it would be better than in this test class.